src/services/photo/index.js

```diff
@@ -169,32 +165,43 @@ class PhotosService {
   }
 
   create(data, params) {
-    return uploadToGCS(params.file)
+    const gcsConfig = this.app.get('gcs');
+
+    return uploadToGCS(params.file, gcsConfig)
     .then((file) => savePhotoMetadata(file))
     .then((photoDoc) => respondWithPhotoMetadata(photoDoc))
     .catch((err) => Promise.reject(err));
   }
 }
 
-
-function uploadSaveRespondByUrl(url) {
-  return uploadToGCSByUrl(url)
+function uploadSaveRespondByUrl(url, gcsConfig) {
+  return uploadToGCSByUrl(url, gcsConfig)
     .then((file) => savePhotoMetadata(file))
     .then((photoDoc) => respondWithPhotoMetadata(photoDoc))
     .catch((error) => Promise.reject(error));
 }
 
 class UploadPhotoFromUrlService {
+  setup(app) {
```

Comment:
Sorry, I am not knowing class in Javascript that well.
Could you explain why we need setup(app)?